packages/repl-cli/DASHBOARD.md

```
@@ -88,7 +88,11 @@ lrepl
 
 ## 🏗️ Architecture
 
+<<<<<<< HEAD
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)